### DISCUSSION TITLE
8EC177BD-0AE1-40EB-AF3B-815304EBF65B

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#74] [General]
# Exhibit-3

Exhibit-3, also known as EXOI-24. Is a (fictional) solar system discovered by the (also fictional) Exhibit North and South Observatories. They both monitored the system as a collaborative effort following the confirmation of Exhibit-2 by Exhibit South, the system was marked as an object of interest after a transit signal of what would later be confirmed as Exhibit-3 d was recorded in August of 2014. Later signals followed that year by Exhibit-3 c and Exhibit-3 b. Over a year new transits were recorded, with Exhibit-3 e and f transiting in 2015, Exhibit-3 g in 2016, and the only transit of Exhibit-3 h in the Summer of 2019. Exhibit-3 i and Exhibit-3 j are considered confirmed by continuous Radial Velocity observations, but it's important to note that they haven't made any transits yet and won't make any for a long time, Exhibit-3 i is predicted to cross Exhibit-3 on Christmas Eve 2035, and Exhibit-3 j in February of 2052. The system is still considered under active study by Exhibit North and South.

https://celestia.mobi/addon?item=8EC177BD-0AE1-40EB-AF3B-815304EBF65B&lang=en

**Comment by exocritic5457** [comment]
Couple things. f's ocean goes from being reflective to non-reflective via a seam, and bits of the ocean don't seem to be reflective at all.

Also, i and j are completely missing a big part of their textures along with the rest of the texture that is there having RGB moiré lines covering it.

**Reply by exocritic5457** [reply]
f seems to have 2 seams splitting a quarter of the surface, easily seen on both of the ice caps as well, and upon further looking it seems that the land and oceans do not line up as some of the landmasses actually appear to reflect the light of the star.

**Reply by Karistus** [reply]
This should've been fixed before uploading, what?

**Reply by Karistus** [reply]
should be updated now

**Reply by exocritic5457** [reply]
f still appears to have some of the same problems, like the bump map being perfectly flat for a quarter of the planet, in between 2 seams, and the rest of the bump map being offset from the planet's texture, resulting in flat land and noticeably bumpy water, something that the similar e correctly doesn't have. i and j are fine, never seen them glitch after the update

**Reply by exocritic5457** [reply]
those are some cool pictures. Excited for the other update

**Comment by exocritic5457** [comment]
I would like to apologize on behalf of my past self. Fictional things with realistic lore concerning their own discovery methods are cool.

**Comment by kmjordan122313-lgtm** [comment]
great revamp update of this addon (^_^)
